dashboard/test/models/peer_review_test.rb

```diff
@@ -22,7 +22,64 @@ class PeerReviewTest < ActiveSupport::TestCase
     level_source = create :level_source, data: 'My submitted answer'
 
     initial = PeerReview.count
-    @user.track_level_progress_async(script_level: @script_level, new_result: 100, submitted: true, level_source_id: level_source.id)
+    @user.track_level_progress_async(script_level: @script_level, new_result: Activity::UNSUBMITTED_RESULT, submitted: true, level_source_id: level_source.id)
     assert_equal PeerReview::REVIEWS_PER_SUBMISSION, PeerReview.count - initial
```

Comment:
Is this guaranteed to work? The message gets enqueued in the progress queue, but is there a guarantee that it gets dequeued before the assert statement?

You might want to try track_level_progress_sync instead